packages/esm-ohri-pmtct-app/src/config-schema.ts

```diff
@@ -18,9 +18,87 @@ export const configSchema = {
       motherPostnatal: '269bcc7f-04f8-4ddc-883d-7a3a0d569aad',
     },
   },
+  obsConcepts: {
+    _type: Type.Object,
+    _description: 'List of observation concept UUIDs related to PMTCT.',
+    _default: {
+      caseID: '162576AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      mnchEncounterType_UUID: '3eb24129-9f7d-4372-a14e-c2cf577e5db1',
+      clientsEnrolledToCare: '51bec6f7-df43-426e-a83e-c1ae5501372f',
+      vLResultsConcept: '1305AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      PTrackerIdentifierType: '4da0a3fe-e546-463f-81fa-084f098ff06c',
+      artUniqueNumberType: '9d6d1eec-2cd6-4637-a981-4a46b4b8b41f',
+      pTrackerIdConcept: '6c45421e-2566-47cb-bbb3-07586fffbfe2',
+
+      //Labour & Delivery
+      labourAndDeliveryEncounterType: '2678423c-0523-4d76-b0da-18177b439eed',
```

Comment:
I think you duplicated all the encounter types. There's already this exact thing called `laborAndDelivery` in the encounter object. Search for occurrences of it and replace them

Reply:
I have removed the duplicates.. as for the naming like `laborAndDelivery` and `labourAndDeliveryEncounterType`, one references  ohri and the other namibia

Reply:
This isn't a good sign. Do u know which forms they're used in?

Reply:
@pirupius.  one encTpes uuid is only being used. Therefore, I have removed the following  encTypes that are not being used in the ptmtct module; 
```
      labourAndDeliveryEncounterType: '2678423c-0523-4d76-b0da-18177b439eed',
      antenatalEncounterType: '2549af50-75c8-4aeb-87ca-4bb2cef6c69a',
      antenatalVisitType: '164181AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
      mchVisitType: '140084BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB',
```